CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+## Table of Contents
+
+- [v0.1.0](#v010)
+
+## v0.1.0 (unreleased)
```

Comment:
Didn't Patryk PR remove this? Or we want for a release to complete?

Reply:
yes, I created the file before Patryk's update. It should be aligned now.